src/nccl_ofi_rdma.c

```diff
@@ -1341,6 +1344,20 @@ static inline int handle_write_comp(struct fi_cq_data_entry *cq_entry, nccl_net_
 	return 0;
 }
 
+/**
+ * @brief	Handle completion for a flush request
+ */
+static inline int handle_flush_comp(nccl_net_ofi_rdma_req_t *req)
+{
+	int ret;
+	nccl_net_ofi_rdma_recv_comm_t *r_comm = (nccl_net_ofi_rdma_recv_comm_t *)req->comm;
+	nccl_net_ofi_rdma_ep_t *ep = (nccl_net_ofi_rdma_ep_t *)r_comm->base.base.ep;
+
+	ret = inc_req_completion(req, 0, ep->num_rails);
```

Comment:
This should be `flush_data->total_num_compls` instead of `ep->num_rails`. Or, we should just remove the former variable, as it's unused.

Reply:
ack, i'd keep `flush_data->total_num_compls` and use it here.